ores/api.py

```diff
@@ -105,10 +107,11 @@ def _score_request(self, context, rev_ids, models):
         logging.debug("Sending score request for {0} revisions"
                       .format(len(rev_ids)))
         start = time.time()
-        response = requests.get(url, params=params,
-                                headers=self.headers,
-                                verify=True, stream=True)
+
         try:
+            response = requests.get(url, params=params,
+                                    headers=self.headers,
+                                    verify=True)
```

Comment:
ValueError should only apply to processing JSON.

Reply:
That's true, lemme back that out.